Package.swift

```diff
@@ -42,7 +42,7 @@ let package = Package(
                 "MapboxNavigationNative",
             ],
             exclude: ["Info.plist"],
-            resources: [.copy("MBXInfo.plist")]),
+            resources: [.copy("Resources/MBXInfo.plist")]),
```

Comment:
Just realized that this will probably break all `Bundle.module.url(for...)` calls, will double check that it still works